libs/libtirpc/patches/800-bzero-to-memset.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/des_impl.c
++++ b/src/des_impl.c
+@@ -588,7 +588,7 @@ _des_crypt (char *buf, unsigned len, str
+     }
+   tin0 = tin1 = tout0 = tout1 = xor0 = xor1 = 0;
+   tbuf[0] = tbuf[1] = 0;
+-  __bzero (schedule, sizeof (schedule));
++  memset (schedule, 0, sizeof (schedule));
```

Comment:
https://github.com/openembedded/openembedded-core/blob/master/meta/recipes-extended/libtirpc/libtirpc/remove-des-functionality.patch
Is that a no go? :-)